reaper-adm-extension/src/reaper_adm/pluginmain.cpp

```diff
@@ -193,10 +86,16 @@ extern "C" {
     auto pluginCallbackHandler = EARPluginCallbackHandler::getInstance();
     rec->Register("API_registerPluginLoad", reinterpret_cast<void*>(&registerPluginLoad));
 
+    auto api = reaper->api();
+
+    UpdateChecker updateChecker;
+    if (updateChecker.getAutoCheckEnabled()) {
+        updateChecker.doUpdateCheck(false, 1000);
```

Comment:
Does this mean there's an additional 1 second delay if you startup offline, or does that return an unreachable error pretty much instantly?

Reply:
No, offline aborts pretty quickly (usually ~100ms)